plugins/source/tailscale/Makefile

```diff
@@ -0,0 +1,36 @@
+# Install tools
+.PHONY: install-tools
+install-tools:
+	@echo Installing tools from tools/tool.go
+	@cat tools/tool.go | grep _ | awk -F'"' '{print $$2}' | xargs -tI % go install %
+
+# Install pre-commit hooks. This requires pre-commit to be installed (https://pre-commit.com/)
+.PHONY: install-hooks
+install-hooks:
+	pre-commit install
+
+# Unit tests
+.PHONY: test
+test:
+	go test -timeout 3m ./...
+
+.PHONY: lint
+lint:
+	@golangci-lint run --config ../../.golangci.yml
+
+.PHONY: gen-code
+gen-code:
+	grep -rl '// Code generated by codegen; DO NOT EDIT.' resources/services/* | xargs rm
+	go run codegen/main.go
+
+.PHONY: gen-docs
+gen-docs:
+	rm -rf ./docs/tables/*
+	go run main.go doc ./docs/tables
+#	sed 's_(\(.*\))_(https://github.com/cloudquery/cloudquery/blob/main/plugins/source/tailscale/docs/tables/\1)_' docs/tables/README.md > ../../../website/pages/docs/plugins/sources/tailscale/tables.md
+#	go run main.go doc --format json docs
+#	rm docs/__tables.json
```

Comment:
These two lines are only necessary for plugins with policies, so we can safely delete them (the policy docs use the JSON output of tables)